### ISSUE TITLE
Redirects don't work with multiple sites

### ISSUE THREAD

**Issue by lee-kagiso** [torchbox/wagtail#1461]
Right now I have two sites configured:

![redirects](https://cloud.githubusercontent.com/assets/11830393/8475179/530810b8-20b6-11e5-8427-97f9e4581f65.png)

I have noticed that redirects don't work properly when you have more than one site, I can't put my finger on the exact issue, other than they don't kick in.

I think it is because I can't specify the site that the rule applies to, and then the moment you have more than one site the redirects module doesn't know if the rule applies to this site or not?

**Comment by kaedroho** [comment]
Thanks @lee-kagiso

Looking at the code behind this (https://github.com/torchbox/wagtail/blob/master/wagtail/wagtailredirects/models.py#L36-L40) you can either have a redirect that applies to all sites or one particular site. When it new redirect is created, it is currently assigned to the site that the editor was using to browse the admin with: https://github.com/torchbox/wagtail/blob/master/wagtail/wagtailredirects/views.py#L112.

I can't remember the reason behind implementing it this way, but I think we should remove that bit of code at least until we have support for selecting the site (#1163), so all new redirects will be global.

A workaround that should get your current redirects working for now (assuming you want all redirects to be global):

```python
>>> from wagtail.wagtailredirects.models import Redirect
>>> Redirect.objects.all().update(site=None)
```

**Comment by lee-kagiso** [comment]
@kaedroho thanks, I never spotted that you have to change the admin url to the site in question.
I guess as a superadmin I thought I could change any sites redirect rules in one spot...

**Comment by gasman** [comment]
Fixed in #1475; making the site field configurable is covered by #1163.

**Comment by lee-kagiso** [comment]
Nice, thanks!
